docs/docs/gatsby-on-windows.md

```diff
@@ -16,3 +16,10 @@ powershell console. On installing this package, it downloads and installs the
 Visual C++ Build Tools 2015, provided free of charge by Microsoft. These tools
 are required to compile popular native modules. It will also install Python
 2.7, configuring your machine and npm appropriately.
+
+## Installing dependencies without troubles.
+
+Even if you love using a pretty terminal as [hyper](https://hyper.is/) with a git bash,
```

Comment:
no need to mention hyper here... Hyper is great but it's irrelevant to using Gatsby. My preferred "voice" for documentation is a "helpful, focused, and neutral".